### PR TITLE
Add patch release schedule for 1.29

### PR DESCRIPTION
**Pull request by jimangel**
I tried my best to modify the patch release schedule for 1.29 according to feedback from previous release cycles and taking into account KubeCon NA in November. Feedback / review welcome!

TL;DR:
| name    | date |
| -------- | ------- |
| 1.29.0-alpha.2   | Tue Oct 10   |
| 1.29.0-alpha.3 (code freeze) | Thur Nov 2    |
| **KCNA** | Nov 6-9    |
| 1.29.0-beta.0    | Thur Nov 16    |
| 1.29.0-rc.0 (and release branch creation)    | Tue Nov 21    |
| **US HOLIDAY** | Thur Nov 23 |
| 1.29.0-rc.1    | Tue Nov 28    |
| 1.29.0    | Tue Dec 5    |

/cc @kubernetes/release-managers 
/cc @Priyankasaggu11929

### CODE REVIEW

File: releases/release-1.29/README.md [hunk 1]
```diff
@@ -48,47 +48,46 @@ The 1.29 release cycle is as follows:
...
+| Major Themes complete                                                                  | Release Notes Lead            | Tuesday 28th November 2023                                                                                            | week 13  |                                                        |
+| Docs complete — All PRs reviewed and ready to merge                                    | Docs Lead                     | Tuesday 28th November 2023                                                                                            | week 13  |                                                        |
+| 1.29.0-rc.1 released                                                                   | Branch Manager                | Thursday 28th November 2023                                                                                           | week 13  |                                                        |
```
Comment (marosset) [inline releases/release-1.29/README.md:86]: Nov 28 is a Tuesday.
Did you mean Nov 28 or Thursday Nov 30?

Comment (jimangel) [reply releases/release-1.29/README.md]: Good catch! Just pushed a fix (intended 11/28/23)

**Review by cpanato — APPROVED** [review]
/approve
/lgtm

**Comment by Priyankasaggu11929** [comment]
Late comment – the timeline looks good to me. Thanks for the PR @jimangel. :pray:

**Pull request merged**